modules/apps/frontend-css/frontend-css-variables-web/src/test/resources/com/liferay/frontend/css/variables/web/internal/theme/css-variables.json

```diff
@@ -0,0 +1,14 @@
+{
+	"variables": {
```

Comment:
I'm also thinking how to process this without equating `object` keys to `css variables`...

```json
{
    "tokens": [
        {
            cssVariable: "background",
            label: "background-color",
            type: "color"
        }, {
            cssVariable: "font",
            label: "font",
            type: "text"
        }
    ]
}
```

Wouldn't this give us some additional room to expand the API in the future without compromising the design? If we rename this to `tokens.json` then it makes it more general and it doesn't feel so constrained to specific `css-variables`.

What do you think?

Reply:
I also thought about making this json file more general, but didn't know how to name it or organize it. 

Maybe we can be ambitious and name it something very generic like `theme.json` so that we can use the same file infrastructure in the future to add any information we think is relevant. 

Then we would name the section something like `css-variables` to leave room for any other data we may add in the future.

Does that make sense?

Reply:
BTW, I don't like `theme.json`, but I cannot think of any good name now. However we can defer that a bit until we forward this pull.

Reply:
Thinking about this after [this reply](https://github.com/liferay-frontend/liferay-portal/pull/17#discussion_r443542647) my suggestion may not apply any more, as we may declare as many new files as we want in the package.json, thus we don't need a single well-known file any more, nor create sections inside it.

Regarding your suggestion (using an array of objects instead of an object of objects) it's OK. I find it a bit more verbose, but it is true that it's more flexible.

And for the name of the file, it wouldn't be very important, because we let users configure it in the package.json file. 

We would still need to decide if we want it to be only for css variables or leave room for something else. This is something I'm not 100% sure of, but I would say that given that we don't have any other use in mind now (at least me) it would be better to make it clear that the file is for css-variables, not a vague term like `tokens`. 

Because if we name it tokens, we are making it more difficult to understand for the final user, adding another term which is not yet defined. And it's true that we may find something else that fits into the `tokens` idea in the future, but it's also very possible that we end up mixing miscellaneous unrelated things in the same file just because they are "tokens".

So unless you have a clear idea in your mind of what we will use the term `tokens` for in the future, I would consider it a premature generalization.